job-server-python/src/test/scala/spark/jobserver/python/SubprocessSpec.scala

```diff
@@ -152,10 +152,17 @@ class SubprocessSpec extends FunSpec with Matchers with BeforeAndAfter with Befo
   )
 
   private def setupPythonProcess(port: String, token: String): scala.sys.process.ProcessBuilder = {
+    // Spark-2.4 does not support python >= 3.8 (see https://github.com/apache/spark/pull/26194) leading to
```

Comment:
+1 for the comment, can you please also make it part of the commit message.

Reply:
I just noticed that you already added this in PR message. This is already enough, i guess.

Reply:
Done